lesson14/js/index.js

```diff
@@ -0,0 +1,83 @@
+const modalBtn = document.getElementById('js-modal-btn');
+const requestBtn = document.getElementById('js-request-btn');
+const modal = document.getElementById('js-modal');
+const ul = document.getElementById('js-ul');
+
+function addLoading() {
+  const wrapper = document.getElementById('js-wrapper');
+  const loading = document.createElement('img');
+  loading.src = "./img/loading-circle.gif";
+  loading.id = "loading";
+  wrapper.appendChild(loading);
+}
+
+function removeLoading() {
+  const loading = document.getElementById('loading');
+  loading.remove();
+}
+
+async function fetchErrorHandling(response) {
+  if (!response.ok) {
+    throw new Error("サーバーエラーが発生しました");
+  } else {
+    return await response.json();
+  }
```

Comment:
```suggestion
  if (response.ok) {
    return await response.json();
  }
  throw new Error("サーバーエラーが発生しました");
```

Reply:
Thank you very much.
It's simpler than using Strict equality in the conditional expression.
I have reviewed the if statement and fixed it. 6fe67a0